src/stratis/stratis.rs

```diff
@@ -2,5 +2,103 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+use std::cell::RefCell;
+use std::collections::HashSet;
+use std::fs;
+use std::os::unix::fs::symlink;
+use std::path::{Path, PathBuf};
+use std::rc::Rc;
+use std::str;
+
+use engine::Engine;
+use stratis::StratisResult;
 
 pub const VERSION: &str = env!("CARGO_PKG_VERSION");
+
+pub const DEV_PATH: &str = "/dev/stratis";
+
+/// Set up directories and symlinks under /dev/stratis based on current
+/// config. Clear out other stuff.
+pub fn setup_devlinks(engine: &Rc<RefCell<Engine>>) -> StratisResult<()> {
+    let mut existing_dirs = HashSet::new();
+    for dir_e in fs::read_dir(DEV_PATH)? {
```

Comment:
The read_dir() causes a failure on one of the Jenkins machines because /dev/stratis doesn't exist.  I thought something was wrong with the test machine - so I looked at it.  Just an FYI.

Reply:
thanks for noticing these issues!